examples/arrakisclient/README.md

```diff
@@ -0,0 +1,89 @@
+A complete version of this example can be found in this repository in [`basic.py`](basic.py)
+
+In Python, we have support for declaring ORM-like models which allow you to easily interact with the data in your tenant.
+Here we define some models for `User` and `Document` which match the tenant namespace configuration.
+
+```py
+from arrakisclient.types.namespace import ArrakisNamespace, Relation
+
+
+# Define some ORM-like models, must match the namespaces configured in the tenant
```

Comment:
Use the new terminology here and add a note of why the classes are named differently

Reply:
This just got moved and I'm not sure it's a totally valid example. Maybe we should just flag everything as arrakisclient as legacy and move on?